### ISSUE TITLE
Out of memory after 5 epoch

### ISSUE THREAD

**Issue by bzdxiaohm** [NoamRosenberg/AutoML#6]

**Comment by NoamRosenberg** [comment]
@bzdxiaohm to what is the alpha_epoch hyper-parameter set to in your code?

**Comment by bzdxiaohm** [comment]
> @bzdxiaohm to what is the alpha_epoch hyper-parameter set to in your code? If it's set to 5 this might be the reason. After a number of epochs the gradients for the alpha and beta parameters begin to be computed. I recommend using a larger GPU or shrinking the model size in some way.

thx a lot，It appeared when I was training with 4 GPU， I will run the code with only 1 GPU

**Comment by NoamRosenberg** [comment]
@bzdxiaohm keep us updated
